BasicSample/Assets/Microsoft.MixedReality.OpenXR.MainSample.asmdef

```diff
@@ -1,6 +1,6 @@
 {
     "name": "Microsoft.MixedReality.OpenXR.MainSample",
-    "rootNamespace": "",
+    "rootNamespace": "Microsoft.MixedReality.OpenXR.Samples",
```

Comment:
it seems we should correct the namespace and assembly name.  should it be "BasicSample" instead of "mainsample" or "samples"?

Reply:
Definitely good with that! I can change both to `BasicSample`. Should I update all the scripts as well? I'll do that in a separate PR.

Reply:
Yeah, "BasicSample" sounds good.  In a separate PR sounds fine as well.

Reply:
Opened #30 for updating the namespace!

Reply:
Didn't we want to change the assembly name as well?
> "name": "Microsoft.MixedReality.OpenXR.MainSample",

Reply:
I removed this change from this PR and included it in #30